app/Photo.php

```diff
@@ -130,9 +130,16 @@ public function prepareData()
 			$photo['license'] = $this->license;
 		}
 
+		if (PhotoFunctions::isValidVideoType($this->type)) {
```

Comment:
@hermzz there should be a way to avoid this static call right?

Reply:
Yeah, why not just `strpos($this->type, 'video') === 0`? If it's been uploaded we already know it's valid, we just need to check if it's a video or image.